rdbms/src/main/java/org/spine3/server/storage/jdbc/util/Serializer.java

```diff
@@ -39,34 +39,37 @@
 @SuppressWarnings("UtilityClass")
 public class Serializer {
 
-    private Serializer() {}
+    private Serializer() {
+    }
 
     /**
      * Serializes a message to an array of bytes.
      *
      * @param message a message to serialize
-     * @param <M> a message type
+     * @param <M>     a message type
      * @return a byte array
      */
     public static <M extends Message> byte[] serialize(M message) {
         checkNotNull(message);
         final Any any = AnyPacker.pack(message);
-        final byte[] bytes = any.getValue().toByteArray();
+        final byte[] bytes = any.getValue()
```

Comment:
Let's do the serialization in the same way we do for GAE. I think, it's done a bit differently there.

That is required so that the app (consisting of several services) could use both relational and no-SQL storages with the same `Message` data format in all storages.